tests/functional/r/regression/regression_6531_crash_index_error.py

```diff
@@ -0,0 +1,28 @@
+# pylint: disable=missing-docstring, redefined-outer-name
```

Comment:
```suggestion
"""Regression test for https://github.com/PyCQA/pylint/issues/6531."""

# pylint: disable=missing-docstring, redefined-outer-name
```

In general I like to add links to regression issues. Only adding the number doesn't make it clear whether it is the issue from `GitHub`, `gitlab` or even the old `logilab` tracker. Especially for some of the lower numbers this is an issue. We don't know where `pylint` might be hosted in ten years time so better save than sorry!

Reply:
Fair point. Sometimes I'm unsure of the convention but in this case I could have been more clear.

Reply:
I think I'm a little too docstring-loving myself, so don't always trust me 😄 But in this case I think there is a real benefit!

Reply:
I completely agree with you. To be honest I rushed the changes a bit last night/today, so I'm not surprised :)